catalog/task/publish-index-image/0.1/tests/run.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: publish-index-image-run-empty-params
+spec:
+  params:
+    - name: requestJsonResults
+      value: ""
+    - name: targetIndex
+      value: ""
+    - name: retries
```

Comment:
You followed my suggestion and removed other optional parameters from the sample/test yamls, let's remove all the optional ones if we are going this route. That includes this parameter